### PR TITLE
Fix issue with deleting buckets using `rmdir`

### PR DESCRIPTION
**Pull request by DerPhysikeR**
Fix issue with deleting buckets using `rmdir` as described in #117.

### CODE REVIEW

File: s3path.py [hunk 1]
```diff
@@ -321,6 +321,8 @@ def rmdir(self, path):
         bucket = resource.Bucket(bucket_name)
         for object_summary in bucket.objects.filter(Prefix=key_name):
             self._boto3_method_with_parameters(object_summary.delete, config=config)
+        if path.is_bucket:
+            bucket.delete()
```
Comment (liormizr) [inline s3path.py:325]: In order to pass relevant kwargs and configurations
Please change the delete call like this:

``` python
self._boto3_method_with_parameters(bucket.delete, config=config)
```

**Review by liormizr — CHANGES_REQUESTED** [review]
Hi @DerPhysikeR 
Looks greate
One comment and we can push this to the next version

File: s3path.py (hunk 1 above)
Comment (DerPhysikeR) [reply s3path.py]: Done. I added the requested change.

**Review by liormizr — APPROVED** [review]
very nice, Tx

**Pull request merged**